### DISCUSSION TITLE
SSO dates!!

### DISCUSSION THREAD

**Discussion by sbminer123** [tobychui/zoraxy#432] [General]
absolutely love this product, amazing job. I really want a SSO solution and I can see its in the pipeline, do you know when this would be happening?

**Comment by tobychui** [comment]
@sbminer123 Hopefully before March 2025 (for the full version), the authelia support is currently hacked together by contributors PR and will be included as an experimental features from next release.

**Reply by sbminer123** [reply]
That's awesome to hear 😊
